galsim/vonkarman.py

```diff
@@ -98,7 +98,7 @@ class VonKarman(GSObject):
     _is_analytic_k = True
 
     def __init__(self, lam, r0=None, r0_500=None, L0=25.0, flux=1, scale_unit=arcsec,
-                 do_delta=False, suppress_warning=False, gsparams=None):
+                 do_delta=False, suppress_warning=False, gsparams=None, _force_stepk=0.0):
```

Comment:
I think we should make this a regular documented parameter, rather than a nominally private one.  After all, you are proposing it for use by an external package (Piff).  Not to enable some internal-to-GalSim feature.